### PR TITLE
FIX highlightColor

### PR DESCRIPTION
**Pull request by ndelangen**
Issue:
screen_shot_2018-12-21_at_10 32 42_am_480

after fix:
screenshot 2019-01-05 at 12 03 04
screenshot 2019-01-05 at 12 01 17

### CODE REVIEW

**Review by Hypnosphi — APPROVED** [review]

**Comment by Hypnosphi** [comment]
I can't spot a difference in failing screenshots. Is it a Chromatic bug?

**Comment by shilman** [comment]
@ndelangen do we need to patch this into `next`?

**Comment by ndelangen** [comment]
yes

File: lib/ui/src/modules/ui/components/stories_panel/stories_tree/tree_decorators.js [hunk 1]
```diff
@@ -114,7 +114,7 @@ function HeaderDecorator(props) {
     };
   }
 
-  const name = highlightNode(node, style);
+  const name = highlightNode(node, style, theme.highlightColor, 'black');
```
Comment (Hypnosphi) [inline lib/ui/src/modules/ui/components/stories_panel/stories_tree/tree_decorators.js]: Can you please make text color customizable here as well? `black` doesn't play well with our `highlightColor` =(
screenshot 2019-01-11 at 20 25 19

**Pull request merged**